package.json

```diff
@@ -27,7 +27,7 @@
     "cf-pagination": "3.1.1",
     "cf-tables": "1.1.0",
     "cf-typography": "3.1.0",
-    "cfpb-chart-builder": "0.0.8",
+    "cfpb-chart-builder": "git://github.com/cfpb/cfpb-chart-builder.git#tick-thinker",
```

Comment:
Garg. That should be the release version. There's a thing I always forget about where `npm install --save` doesn't seem to work with shrinkwrapping or something. I'll fix it.